### PR TITLE
Add stylish D-Man

### PR DESCRIPTION
**Pull request by WalterBright**
Another masterpiece created with Microsoft Paint!

### CODE REVIEW

File: dstyle.dd [hunk 1]
```diff
@@ -240,72 +266,36 @@ $(P
...
+return trustedCast!RetType(result); 
+---
+
+    $(P This requires review of what $(CODE RetType) and $(CODE result) are to determine
+    whether it is actually safe or not.)
 )
 
 Macros:
     TITLE=The D Style
+    WIKI=DStyle
```
Comment (wilzbach) [inline dstyle.dd]: this macro isn't used anymore

File: dstyle.dd [hunk 2]
```diff
@@ -139,35 +164,36 @@ ubyte asciiChar;
...
     $(P ... is preferred over ...)
-
+    
 -------------------------------
 alias uint size_t;
 -------------------------------
 
     $(P ... because ...)
-
+
```
Comment (wilzbach) [inline dstyle.dd]: huh?

**Comment by CyberShadow** [comment]
You'll need to add the image to the makefile (around line 132).

**Comment by WalterBright** [comment]
done

**Pull request merged**